portal/src/components/FilterCategories/FilterDropdown.vue

```diff
@@ -36,38 +39,99 @@ export default {
       type: Array,
       default: Array,
     },
+    state: {
```

Comment:
Can we make this isVisible or similar? State sounds like some object gets passed into it and which heavily influences a render. This just seems to go into a v-show as a Boolean. Or am I missing something?